stl/inc/flat_map

```diff
@@ -0,0 +1,300 @@
+#pragma once

```

Comment:
Please use the standard include guard mechanism. `#pragma once` is a bit buggy at this moment, see #3895.